lib/elements/rebar.rb

```diff
@@ -0,0 +1,66 @@
+require './db/base'
+require './lib/materials/steel'
+
+module Elements
+  class Rebar < Materials::Steel
```

Comment:
I don't think `Rebar` should inherit from `Materials::Steel`. What happens if later on, we want to support rebars from other materials? I think this could be better modeled by defining a `@material` attribute inside rebar, which would be steel in most cases

I think `Rebar` should inherit from a `Elements::Base` class, which would contain nothing for now. Similar to `Materials::Base`

What do you think?

Reply:
Agree, and Rebar isn't another material, so It should not inherit from any Material